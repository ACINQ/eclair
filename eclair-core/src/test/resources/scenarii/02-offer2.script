# Simple test that we can commit two HTLCs
# Initial state: A=1000000 sat, B=1000000 sat, both fee rates=10000 sat
A:offer 1000000,7b3d979ca8330a94fa7e9e1b466d8b99e0bcdea1ec90596c0dcc8d7ef6b4300c
A:offer 2000000,6016bcc377c93692f2fe19fbad47eee6fb8f4cc98c56e935db5edb69806d84f6
A:commit
B:recvoffer
B:recvoffer
B:recvcommit
A:recvrevoke
B:commit
A:recvcommit
B:recvrevoke
checksync
echo ***A***
A:dump
echo ***B***
B:dump

