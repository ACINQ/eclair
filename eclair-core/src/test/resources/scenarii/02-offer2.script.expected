***A***
LOCAL COMMITS:
 Commit 1:
  Offered htlcs: (0,1000000 msat) (1,2000000 msat)
  Received htlcs:
  Balance us: 997000000 msat
  Balance them: 1000000000 msat
  Fee rate: 10000
REMOTE COMMITS:
 Commit 1:
  Offered htlcs:
  Received htlcs: (0,1000000 msat) (1,2000000 msat)
  Balance us: 1000000000 msat
  Balance them: 997000000 msat
  Fee rate: 10000
***B***
LOCAL COMMITS:
 Commit 1:
  Offered htlcs:
  Received htlcs: (0,1000000 msat) (1,2000000 msat)
  Balance us: 1000000000 msat
  Balance them: 997000000 msat
  Fee rate: 10000
REMOTE COMMITS:
 Commit 1:
  Offered htlcs: (0,1000000 msat) (1,2000000 msat)
  Received htlcs:
  Balance us: 997000000 msat
  Balance them: 1000000000 msat
  Fee rate: 10000
